packages/react-tv-cli/scripts/tizen/run.js

```diff
@@ -0,0 +1,177 @@
+const path = require('path');
+const fs = require('fs-extra');
+const chalk = require('chalk');
+const execSync = require('child_process').execSync;
+
+function defaultCLIEnv() {
+  //return '/opt/tizen/tools/ide/bin';
+  return 'E:/Ferramentas/tizen/tools/ide/bin';
+}
+
+function getPackageId(root) {
+  const appinfo = path.resolve(root, 'react-tv/tizen/config.xml');
+  const content = fs.readFileSync(appinfo, {encoding: 'utf-8'});
+
+  const re = new RegExp(/tizen:application id="(.*?)"/);
+  const matches = content.match(re);
+
+  if (!matches) {
+    return null;
+  }
+
+  return matches[1];
+}
+
+function isReactTVTizenProject(root) {
+  const appinfo = path.resolve(root, 'react-tv/tizen/config.xml');
+  if (fs.existsSync(appinfo)) {
+    return true;
+  }
+  return false;
+}
+
+function run(root) {
+  let tizen_CLI_ENV = process.env.TIZEN_CLI || false;
+  if (!tizen_CLI_ENV) {
+    tizen_CLI_ENV = defaultCLIEnv();
+  }
+
+  process.env.PATH = `${tizen_CLI_ENV}:${process.env.PATH}`;
+
+  if (!isReactTVTizenProject(root)) {
+    const msg = `This project isn\'t a React-TV Tizen Project:
+        Just run "react-tv init"`;
+    return console.log(chalk.dim('[react-tv]'), msg);
+  }
+
+  const packageJson = require(path.resolve(root, 'package.json'));
+  const ReactTVConfig = packageJson['react-tv'];
+  if (!ReactTVConfig) {
+    return console.log(
+      chalk.dim('[react-tv]'),
+      'You should set react-tv properties on package.json'
+    );
+  }
+
+  if (!ReactTVConfig.files || !ReactTVConfig.files.length) {
+    return console.log(chalk.dim('[react-tv]'), 'You should add files');
+  }
+
+  // TODO: option to create/select profiles?
```

Comment:
The user could pass it by a CLI argument or he could be asked interactively here.